grapher.py

```diff
@@ -68,12 +81,50 @@ def find_anomalies(self, random_data):
                 anomalies.append(outlier)
         return anomalies
 
+# User will enter a number from 1-7 based on what they want to do
+def choose_commands():
+    print("What commands would you like to do for stock ticker you chose?")
+    select = input("Please enter the number ' 1, 2, 3, 4, 5, 6, 7'\n"
+                + "1. Print the Bar Graph\n"
+                + "2. Print the Line Graph\n"
+                + "3. Print Summary\n"
+                + "4. Print Bar Graph and Summary\n"
+                + "5. Print Line graph and Summary\n"
+                + "6. Print Bar Graph and Line Graph\n"
+                + "7. Print all three\n"
+                )
+    return select
+
+# Output what the user wants
+def determine_outputs(select):
+    if select == "1":
+        main.barGraph()
+    elif select == "2":
+        main.lineGraph()
+    elif select == "3":
+        main.printData()
+    elif select == "4":
+        main.barGraph()
+        main.printData()
+    elif select == "5":
+        main.lineGraph()
+        main.printData()
+    elif select == "6":
+        main.barGraph()
+        main.lineGraph()
+    elif select == "7":
+        main.barGraph()
+        main.lineGraph()
+        main.printData()
+    else:
+        print("Not a valid command")
 
 # setup
-main = mainObj()
 
-# commands
-data = main.getData("KODK")
-# main.printData(data)
-# main.barGraph(data)
-main.lineGraph(data)
+# Allow the user to specify what stock ticker they want to look at when the script is ran
+stock_ticker = input("Please enter FINRA stock ticker Symbol(i.e. Tesla > TSLA, Apple > AAPL)\n")
```

Comment:
It'll be great to take a list of tickers to run instead of a single one. Also would love to see the same menu in the `market_scanner.py`. That way I could let's say pass a list of tickers compiled from Finviz that have let's say price < $5 and float < 20M (500 tickers) which means we could be running the script in less than 1 minute instead of 1 hour that takes now. Maybe I can look into adding something like this but I'll be slow since it's my first time looking at Python